lightning/src/chain/keysinterface.rs

```diff
@@ -64,8 +65,8 @@ pub enum SpendableOutputDescriptor {
 	DynamicOutputP2WSH {
 		/// The outpoint which is spendable
 		outpoint: OutPoint,
-		/// The secret key which must be used to sign the spending transaction
-		key: SecretKey,
+		/// Per commitment point to derive delayed_payment_key by key holder
```

Comment:
derive *how*? Please point the user to ln::chan_utils::derive_private_key, make it pub, and describe exactly which inputs should be passed to it. Based on the docs here I don't know how to sign for this transaction, whereas previously I easily could. Note that the docs also, above, say "The private key which should be used to sign the transaction is provided" which is also wrong now.

Reply:
Done, also made public get_revokable_redeemscript, you do want to regenerate script for verification.